source/unified-test-format/tests/invalid/expectedCommandEvent-commandFailedEvent-hasServerId-type.yml

```diff
@@ -0,0 +1,16 @@
+description: expectedEvent-commandFailedEvent-hasServerId-type
```

Comment:
Prefix should be `expectedCommandEvent-`.

I think you'll need to do this for the renamed files as well.